applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -0,0 +1,118 @@
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        - survey: BLOCK-T429  # Moonlight
+          pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Isr-cal.yaml']
+        # Manual observations during SV blocks?
+        - survey: BLOCK-T364
+          pipelines: []
+        - survey: BLOCK-T365
+          pipelines: []
+        - survey: BLOCK-T366
+          pipelines: []
+        - survey: BLOCK-T367
+          pipelines: []
+        - survey: BLOCK-T368
+          pipelines: []
+        - survey: BLOCK-T369
+          pipelines: []
+        - survey: BLOCK-T373
+          pipelines: []
+        # Miscellaneous scripts, not always images
+        - {survey: "", pipelines: []}
+        # Ignore unknown events
+        - {pipelines: []}
+      preprocessing: |-
+        # TODO: run if and only if ApPipe is a possibility
+        # - survey: BLOCK-T407
+        #   pipelines: ['${PROMPT_PROCESSING_DIR}/pipelines/LSSTCam/Preprocessing.yaml']
+        - {survey: "", pipelines: []}
+        # Don't preprocess anything unknown
+        - {pipelines: []}
+    # TODO: need to adjust this based on observed slew accuracy
+    preloadPadding: 50
+    calibRepo: s3://rubin-summit-users
+
+  s3:
+    imageBucket: rubin-summit
+    endpointUrl: https://sdfembs3.sdf.slac.stanford.edu
+
+  raw_microservice: http://172.24.5.158:8080/presence
+
+  imageNotifications:
+    kafkaClusterAddress: prompt-processing-2-kafka-bootstrap.kafka:9092
+    topic: rubin-summit-notification
+    # TODO: need to adjust this based on observed nextVisit lead time (depends on scheduler) and PP prep time
+    # The shorter this is, the less capacity is wasted on canceled visits
+    imageTimeout: 120
+
+  apdb:
+    config: s3://rubin-summit-users/apdb_config/cassandra/pp_apdb_lsstcam.yaml
+
+  alerts:
+    username: kafka-admin
+    server: usdf-alert-stream-dev.lsst.cloud:9094
+    topic: lsst-alerts
+
+  # TODO: may need to override for debugging
+  logLevel: timer.lsst.activator=DEBUG timer.lsst.daf.butler=DEBUG lsst.diaPipe=VERBOSE lsst.rbClassify=VERBOSE lsst.daf.butler=VERBOSE
+
+  sasquatch:
+    # TODO: production Sasquatch not yet ready
+    endpointUrl: https://usdf-rsp-dev.slac.stanford.edu/sasquatch-rest-proxy
+    namespace: lsst.prompt.prod
+    auth_env: false
+
+  keda:
+    minReplicaCount: 3
+    # TODO: this is scaled for SingleFrame, with 25% margin
+    maxReplicaCount: 1100
+    # TODO: may need to override for debugging
+    # failedJobsHistoryLimit: 100
+
+    # TODO: may need to reduce if we don't have enough capacity
+    # redisStreams:
+    #   expiration: 600
+
+  initializer:
+    # 6 retries is not enough time to fix, e.g., DB permissions problems
+    retries: 15
+    # Keep around for first few days so we can check in the morning PDT
+    cleanup_delay: 21600  # 6 hours
+
+    podAnnotations: {
+      edu.stanford.slac.sdf.project/usdf-embargo: "true"
+    }
```

Comment:
Just to confirm that whether we need this for initializer to use embargo annotation. 

I would have thought that the lines below (116-118) could mean the whole application has it, but I actually don't know.

Reply:
No, the annotations are independent. Pod annotations are mostly used to control behavior, and the init job and Keda behave very differently.

Reply:
As for whether it should be embargo from a policy standpoint... it's not touching embargoed pixels, but it is reading/writing the embargo repo. 🤷

Reply:
Oh, we might as well just use embargo nodes then...